docs/netdata-cloud/node-membership-rules.md

```diff
@@ -0,0 +1,69 @@
+# Node Membership Rules
+
+## Overview
+
+The Node Membership feature provides automatic node assignment to rooms based on user-defined rules. Users can create multiple rules per room to control node membership. When a node in the space matches these rules, it will be automatically included in or excluded from the room where the rules are defined. This allows for dynamic and automated organization of nodes based on their characteristics.
+
+## Rule structure
+
+A node membership rule defines whether a node should be included or excluded from a room based on its host labels. Rules are composed of the following elements:
+
+- `Action`: (Required) It determined the rule's behavior and has the following possible values:
+  - `Included`: (Default) Any node that matches the current rull will be included in the room.
+  - `Excluded`: Any node that matches the current rull will not be included in the room.
+- `Clauses`: (Required) A collection of conditions that must **all be satisfied**. At least one condition is required.
+- `Description`: (Optional) A description of the rule.
+
+### Clauses
+
+Each clause represents a condition that must be met and is consisted of three parts:
+
+- `Label`: The name of the host label to evaluate.
+- `Operator`: The comparison operator between the label and the value.
+- `Value`: The value to compare against.
+
+All three parts of clause must be specified to consider this clause valid.
+
+**Example of a valid clause**
+
+```
+[_architecture] [equals] [x86_64]
+```
+
+## Rules evaluation
+
+Rules are evaluated in a specific order to determine node membership. `Exclude` rules are evaluated first and then `Include` rules do. This means that an `Include` rule takes priority over an `Exclude` rule with the same clauses.
```

Comment:
I think you meant it the other way around: 

```
This means that an EXCLUDE rule takes priority over an INCLUDE rule with the same clauses.
```

Reply:
I updated this, after testing it on the UI, the exclusion is honored it seems, so this sentence was wrong